stl/inc/algorithm

```diff
@@ -1786,6 +1786,39 @@ template <class _ExPo, class _FwdIt, class _Pr, _Enable_if_execution_policy_t<_E
 _NODISCARD _FwdIt remove_if(_ExPo&& _Exec, _FwdIt _First, const _FwdIt _Last, _Pr _Pred) noexcept; // terminates
 #endif // _HAS_CXX17
 
+// FUNCTION TEMPLATE _Erase_nodes_if
+template <class _Container, class _Pr>
+void _Erase_nodes_if(_Container& _Cont, _Pr _Pred) { // erase each element satisfying _Pred
+    auto _First      = _Cont.begin();
+    const auto _Last = _Cont.end();
+    while (_First != _Last) {
+        if (_Pred(*_First)) {
+            _First = _Cont.erase(_First);
+        } else {
+            ++_First;
+        }
+    }
+}
+
+
+// FUNCTION TEMPLATE _Erase_remove_if
+template <class _Container, class _Pr>
+void _Erase_remove_if(_Container& _Cont, _Pr _Pred) { // erase each element satisfying _Pred

```

Comment:
I know this is what is in the standard, but the return type of `void` is antithetical to the "dont throw away information" theme that has been popular in the last release. Especially in this algorithm it would be more or less free

I assume the ship has sailed for C++20.

Reply:
> I assume the ship has sailed for C++20

It's almost certainly too late. There is a proposal to change these (WG21-P1115), but it hasn't yet been approved by WG21.

Reply:
But given https://github.com/cplusplus/nbballot/issues/228 and https://github.com/cplusplus/nbballot/issues/231, maybe there's still chance.

Reply:
If LWG chooses to accept every "review and merge this proposal that LWG didn't have time to review and merge before the CD" National Body comment, another week's worth of meeting time will have to magically appear somewhere in the next four months. This paper is small, but also proposes a change WG21 could make just as easily *after* shipping C++20. I won't hold my breath.

Reply:
This one's my fault - I knew about the inconsistency but didn't find the time to fix it. In any event, there's no action for @SuperWig to take here.